pkg/roachpb/data.go

```diff
@@ -2783,3 +2783,6 @@ func (tid *TenantID) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return unmarshal(tid)
 	}
 }
+
+// SafeValue implements the redact.SafeValue interface.
+func (LeaseAcquisitionType) SafeValue() {}
```

Comment:
nit: small preference to move this method right below `(LeaseSequence).SafeValue`, but this is also fine.

Reply:
Moved!